terraform/modules/core/core.tf

```diff
@@ -210,8 +210,7 @@ resource "aws_alb_listener" "default" {
   depends_on = ["aws_alb_target_group.alb_target_group"]
 }
 
-resource "aws_lb_listener_rule" "redirect_old_teachingjobs_http_traffic" {
-  count = "${var.redirect_old_teachingjobs_traffic}"
+resource "aws_lb_listener_rule" "redirect_all_http_requests_to_https" {
   listener_arn = "${aws_alb_listener.default.arn}"
 
   action {
```

Comment:
@Stretch96 keeping the redirect host here ensures that all traffic is redirected to the right hostname, as you mentioned that someone might directly visit the load balancer and bypass CloudFront.

Reply:
![screenshot 2019-01-21 at 17 00 36](https://user-images.githubusercontent.com/912473/51488718-2e47b380-1d9e-11e9-9ef7-544751f146b5.png)